Gemfile

```diff
@@ -99,4 +99,5 @@ end
 
 group :test do
   gem 'poltergeist'
+  gem 'selenium-webdriver'
```

Comment:
I had 4 remaining failing cuke tests that will (so far) only run successfully when using Selenium.